lib/squid/axis.rb

```diff
@@ -29,7 +29,11 @@ def labels
     end
 
     def width
-      @width ||= labels.map{|label| label_width label}.max || 0
+      @width ||= if val = labels.map{|label| label_width label}.max
+        [val, 70].min
+      else
+        0
+      end
```

Comment:
Vi har brug for lidt mere plads til at skrive Likert-labels på y-aksen, men squid begrænser ikke længden som standard, så grafen kan ende med at blive skubbet helt ud til højre. Dette sørger for at labels aldrig tager mere end 70 pixels bredde.

Reply:
Super 👍